web-marketplace/src/components/organisms/UserSellOrders/hooks/useFetchSellOrders.ts

```diff
@@ -1,5 +1,9 @@
 import { useCallback, useMemo } from 'react';
-import { useQuery, useQueryClient } from '@tanstack/react-query';
+import {
+  QueryObserverOptions,
```

Comment:
this is unused
I guess the easiest is to revert the whole file back to its state on `dev`